tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>we can change the aspect ration of the PV diagram by giving matplotlib an explicit figsize.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
clarified